ci/actionstrings/regenerate-8.13-plus.sh

```
@@ -0,0 +1,72 @@
+
+TOTAL=$(echo "$VERSIONS" | wc -l | tr -d ' ')
+COUNT=0
+
+for VERSION in $VERSIONS; do
+  COUNT=$((COUNT + 1))
+  FILENAME="$DEST_DIR/action_strings_es${VERSION}.txt"
+  OLD_COUNT=$(wc -l < "$FILENAME" | tr -d ' ')
+
+  echo "[$COUNT/$TOTAL] Regenerating $VERSION (was $OLD_COUNT strings)..."
+
+  TEMPDIR=$(mktemp -d -t ror-regen-es-$VERSION-XXXXXXXXXX)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)